clang/unittests/Interpreter/IncrementalCompilerBuilderTest.cpp

```diff
@@ -37,6 +37,9 @@ TEST(IncrementalCompilerBuilder, SetCompilerArgs) {
 }
 
 TEST(IncrementalCompilerBuilder, SetTargetTriple) {
+#ifdef __EMSCRIPTEN__
+  GTEST_SKIP() << "Test fails for Emscipten builds";
+#endif
```

Comment:
Hmm, we should be able to fetch the target triple. 

Can we try ?
```
#ifdef __EMSCRIPTEN__
  const char* triple = "wasm32-unknown-emscripten";
  const char* expected = "wasm32-unknown-emscripten";
#else
  const char* triple = "armv6-none-eabi";
  const char* expected = "armv6-unknown-none-eabi";
#endif
```
here and use `triple` and `expected` below ?